tests/flow/test_index_updates.py

```diff
@@ -176,3 +190,30 @@ def test06_remove_indexed_prop(self):
         expected_result = []
         self.env.assertEquals(result.result_set, expected_result)
 
+    def populate_graph_many_nodes(self, graph):
+        global node_ctr
```

Comment:
Why do we need this variable ?

Reply:
I simplified this, but I don't see reason for it not to remain for future usage.